### PR TITLE
different text for different type of users

### PR DESCRIPTION
**Pull request by projectr495836**
On the side bar, when user is not challenger, it will show "take the challenge". When user is a challenger, it will show "my challenge".

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -27,38 +35,81 @@ function Header() {
...
-    },
-  ];
+  var SidebarData;
+  var flagForTestNonChallenger=0;// make it non 0 if want to test the case for User Type is not challenger
+  if(getUserType()=="challenger" & flagForTestNonChallenger==0){
+
+    SidebarData = [
+      {
+        title: "My Challenge",
+        path: "/",
```
Comment (Devanshusp) [inline src/components/Header/Header.js]: All of these lead to the homepage "/" (placeholders from before)
I think we have most of the pages built out now and can start filling it out, ex. "My Challenge" is /progress

**Review by Devanshusp — CHANGES_REQUESTED** [review]
Hey @projectr495836, nice work updating the sidebar!

I think we can go ahead and fill out the links for each page; the notifications link has also been removed from the most recent design.
Also, lets see if we can follow the finalized design on #23

**Comment by wcouturiaux** [comment]
@projectr495836 is this PR for issue #117?

**Comment by Devanshusp** [comment]
> @projectr495836 is this PR for issue #117?

Yeah

**Pull request closed without merging**